docs/guides/external-mgs/readme.md

```diff
@@ -0,0 +1,151 @@
+---
+authors: Matt Richerson <matthew.richerson@hpe.com>
+categories: provisioning
+---
+
+# Lustre External MGS
+
+## Background
+
+Lustre has a limitation where only a single MGT can be mounted on a node at a single time. In the NNF cluster, this means the total number of Lustre file systems is limited to the number of NNF nodes. It also places extra scheduling constraints on the WLM to ensure that the MGTs don't conflict.
```

Comment:
Don't confuse MGS (the node) with MGT (a piece of storage).

A node can be an MGS, and you can't use that same node for another MGS.

Multiple MGTs can be mounted on that node to be used with that same MGS.

Reply:
I don't think this is true. You're not able to mount multiple MGTs on the same node. 

See the last paragraph here: https://wiki.lustre.org/Lustre_Management_Service_(MGS)